server/otlp/ingester.go

```diff
@@ -16,21 +16,32 @@ import (
 	v1 "go.opentelemetry.io/proto/otlp/trace/v1"
 )
 
+type Ingester interface {
```

Comment:
is this change required? looks like no other package is referencing the interface. maybe I'm missing something though

Reply:
after I deleted the `collector.ingester` interface, we needed this to be exported.